api/include/opentelemetry/metrics/sync_instruments.h

```diff
@@ -97,14 +97,14 @@ class Histogram : public SynchronousInstrument
   /**
    * Records a value.
    *
-   * @param value The increment amount. May be positive, negative or zero.
+   * @param value The increment amount. MUST be non-negative.
```

Comment:
The comment also says "The increment amount", which sounds like a copy and paste from counters, and does not apply to histograms.

In my understanding:

For an histogram, the metric value should be passed, and will decide which bucket stat to increment.

There is no such thing as giving an "increment amount".

Reply:
Good point. Have updated this now.